src/components/site/LogoType.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const LogoType = props => 
+<div class="logotype">
+    <a href="#">
```

Comment:
The `href` should point to the root of any site, which is `/`. We also want to add a title to the link of "View the Baltimore County Government homepage."

Reply:
So without having a better way to do this. I think we need to remove the div. Make the `a` have a class of "logo-type" and adjust accordingly.

Here is the html that i goo to work in  the browser.


```html
<a href="#" class="logotype" style="
    display: block;
    padding: 0 40px;
    text-decoration: none;">
        <span>Baltimore County</span>
        <span>Government</span>
</a>
```

Note the styles would convert to sass in this project.

But the result is...

![image](https://user-images.githubusercontent.com/1933400/64360495-9b7c2200-cfd8-11e9-9934-04a05d5db508.png)

Reply:
@danfox01 confirmed the h1 is actually not needed here, despite existing in the current site.

Reply:
this has been fixed